packages/app/ui/modal/ModalFooter.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+
+import { ReactNode } from "react";
+
+interface ModalFooterProps {
+  children: ReactNode;
+}
```

Comment:
can't understand why this would make it easier

```js
"use client";

import { ReactNode, FC } from "react";

interface ModalFooterProps {
  children: ReactNode;
}

export const ModalFooter: FC<ModalFooterProps> = ({ children }) => {
  return (
    <div className="flex items-center px-4 py-5 space-x-2">{children}</div>
  );
};

```

Reply:
Can't understand why this would make it easier. What am I missing?

```js
"use client";

import { ReactNode, FC } from "react";

interface ModalFooterProps {
  children: ReactNode;
}

export const ModalFooter: FC<ModalFooterProps> = ({ children }) => {
  return (
    <div className="flex items-center px-4 py-5 space-x-2">{children}</div>
  );
};

```